### PR TITLE
DOC-5355: Metrics restructured

### PR DESCRIPTION
**Pull request by AnnaK-R3**

### CODE REVIEW

File: content/en/platform/corda/5.0/deploying-operating/observability/metrics/db-worker.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+own dedicated database). As a result, the activities carried out within the database worker pertain to the database.
+
+The database worker is responsible for handling and serving persistence requests originating from various Corda worker types,
+such as the flow worker or the MGM worker. The flow persistence requests metrics presented in this section measure:
+
+* The time taken to handle the flow persistence requests.
+* The time the flow persistence requests remained on Kafka, from the moment they were added by flows until they were received
+  by the database worker (Kafka lag).
+
+The ledger persistence requests metrics measure the time taken to handle the ledger persistence requests.
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0/deploying-operating/observability/metrics/db-worker.md:26]: Think I commented before that I found this hard to read

File: content/en/platform/corda/5.0/deploying-operating/observability/metrics/db-worker.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+| `corda_ledger_persistence_time_seconds` | Timer | <ul><li>`flowId`</li><li>`ledger_type`</li><li>`operation_name`</li></ul> | The time it takes to execute ledger transaction database request against the database. |
+| `corda_membership_persistence_transaction_time_seconds` | Timer | <ul><li>`operation_name`</li><li>`group`</li><li>`virtualnode`</li></ul> | The time it takes to execute membership persistence transactions. Excludes time spent acquiring a database connection. |
+| `corda_membership_persistence_handler_time_seconds` | Timer | <ul><li>`operation_name`</li><li>`group`</li><li>`virtualnode`</li></ul> | The time it takes to execute membership persistence handlers. Includes time taken to get database connection and execute the transaction. |
+| `corda_db_reconciliation_run_time_seconds` | Timer | <ul><li>`reconciliation_reconciler_type`</li><li>`reconciliation_outcome`</li></ul> | The time needed for a full reconciliation run. |
+| `corda_db_reconciliation_records_count` | Counter | <ul><li>`reconciliation_reconciler_type`</li><li>`reconciliation_outcome`</li></ul></ul> | The number of reconciled records for a reconciliation run. |
+
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0/deploying-operating/observability/metrics/db-worker.md:67]:
```suggestion
```

File: content/en/platform/corda/5.0/deploying-operating/observability/metrics/db-worker.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+Tags:
+* `entityRequest.type`: The type of persistence request.
+* `entityRequest_outcome`: The outcome of processing a request (SUCCESS, FAILURE).
+* `flowId`: The flow ID to correlate with the flow.
+* `ledger_type`: It can be UTXO or CONSENSUAL.
+* `operation_name`: The MGM persistence request name/type.
+* `group`: The membership group within which peer-to-peer communication happens.
+* `virtualnode`: The virtual node identity.
+* `reconciliation_reconciler_type`: The type of reconciler that run, for example, CPI metadata, virtual node metadata.
+* `reconciliation_outcome`: The outcome of a reconciliation run (SUCCESS, FAILURE).
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0/deploying-operating/observability/metrics/db-worker.md:77]:
```suggestion
* `reconciliation_outcome`: The outcome of a reconciliation run (SUCCESS or FAILURE).
```

File: content/en/platform/corda/5.0/deploying-operating/observability/metrics/backing-store.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+| `corda_uniqueness_backingstore_transaction_attempts` | DistributionSummary | <ul><li>`virtualnode_source`</li></ul> | 	The number of attempts that were made before a transaction ultimately succeeded. Generally, this should return 1. In the event that a transaction was unsuccessful due to reaching the maximum number of attempts, this metric is not updated and the failure would be reflected in the `corda_uniqueness_backingstore_transaction_error_count` metric. |
+| `corda_uniqueness_backingstore_db_commit_time_seconds` | Timer | <ul><li>`virtualnode_source`</li></ul> | The time taken by the backing store to commit a transaction (that is, write) to the database. Only updated if data is written to the database, so it is not cumulative across retry attempts for a given transaction. |
+| `corda_uniqueness_backingstore_db_read_time_seconds` | Timer | <ul><li>`virtualnode_source`</li><li>`operation_name`</li></ul> | The time taken to perform a single read operation from the database. If a transaction is retried, each retry contributes independently to this metric, meaning the number is not cumulative across retries. |
+
+Tags:
```
[the remark this answers is not in the document]
Comment (AnnaK-R3) [reply content/en/platform/corda/5.0/deploying-operating/observability/metrics/backing-store.md:51]: leaving as is

File: content/en/platform/corda/5.0/deploying-operating/observability/metrics/crypto-worker.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+| `corda_entity_manager_factory_creation_time_seconds` | Timer | <ul><li>`tenant`</li></ul> | The time taken to create entity manager factories. |
+| `corda_crypto_sign_time_seconds` | Timer | <ul><li>`signature_spec`</li></ul> | The time taken for crypto signing. |
+| `corda_crypto_sigining_key_lookup_time_seconds` | Timer | <ul><li>`lookup_method`</li></ul> | The time taken for crypto signing key lookup. |
+| `corda_crypto_signing_repository_get_instance_time_seconds` | Timer | <ul><li>`tenant`</li></ul> | The time taken to get crypto signing repository instances. |
+| `corda_crypto_get_owned_key_record_time_seconds` | Timer | <ul><li>`operation_name`</li><li>`publickey_type`</li></ul> | The time taken to look up tenant’s owned keys. |
+| `corda_crypto_cipher_scheme_time_seconds` | Timer | <ul><li>`operation_name`</li></ul> | The time taken for crypto cipher scheme encoding and decoding operations. |
+| `corda_crypto_signature_spec_time_seconds` | Timer | <ul><li>`operation_name`</li></ul> | The time taken for crypto signature spec operations including deserializing wire objects to signature spec and vice versa. |
+
+Tags:
```
[the remark this answers is not in the document]
Comment (AnnaK-R3) [reply content/en/platform/corda/5.0/deploying-operating/observability/metrics/crypto-worker.md:71]: leaving as is

File: content/en/platform/corda/5.0/deploying-operating/observability/metrics/_index.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+  podMonitor:
+    enabled: true
+    labels:
+      release: [RELEASE_NAME]
+```
+
+### Exported Metrics
+
+The following Corda-specific metrics are exported and they have been added at the following levels:
+
```
[the remark this answers is not in the document]
Comment (AnnaK-R3) [reply content/en/platform/corda/5.0/deploying-operating/observability/metrics/_index.md:55]: Forgot about {{< childpages >}}. Thanks!

File: content/en/platform/corda/5.0/deploying-operating/observability/metrics/db-worker.md (hunk 1 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/5.0/deploying-operating/observability/metrics/db-worker.md]: updated

**Pull request merged**